td-reproducible-tool/Cargo.toml

```diff
@@ -15,4 +15,5 @@ default = []
 [dependencies]
 argparse = "0.2.2"
 zeroize = "1.5.4"
-regex = "1"
\ No newline at end of file
+regex = "1"
+codec = { git="https://github.com/jyao1/rust-spdm", default-features = false }
```

Comment:
Please dont depend on this.

Reply:
> Please dont depend on this.

Dependency is now removed in latest code change.